src/ingest_validation_tools/table-schemas/assays/gcms-v1.yaml

```diff
@@ -0,0 +1,116 @@
+fields:
+# include: ../includes/vA/all_assays.yaml
+- name: assay_category
+  constraints:
+    enum:
+      - mass_spectrometry_imaging
+- name: assay_type
+  constraints:
+    enum:
+      - GC-MS
+- name: analyte_class
+  constraints:
+    enum:
+      # TODO: Definitely not protein?
+      - metabolites
+      - lipids
+# include: ../includes/v0/is_targeted.yaml
+
+# Level 2:
+
+# include: ../includes/v0/acquisition_instrument.yaml
+- name: ms_source
+  description: The ion source type used for surface sampling.
+  constraints:
+    enum:
+      - MALDI
+      - MALDI-2
+      - DESI
+      - SIMS
+      - nESI
+- name: polarity
+  description: The polarity of the mass analysis (positive or negative ion modes)
+  constraints:
+    enum:
+      - negative ion mode
+      - positive ion mode
+- name: mz_range_low_value
+  description: The low value of the scanned mass range for MS1. (unitless)
+- name: mz_range_high_value
+  description: The high value of the scanned mass range for MS1. (unitless)
+
+- name: mass_resolving_power
+  description: The MS1 resolving power defined as m/∆m where ∆m is the FWHM for a given peak with a specified m/z (m). (unitless)
+  type: number
+- name: mz_resolving_power
+  description: The peak (m/z) used to calculate the MS1 resolving power.
+  type: number
+- name: ion_mobility
+  description: 'Specifies whether or not ion mobility spectrometry was performed and which technology was used. Technologies for measuring ion mobility: Traveling Wave Ion Mobility Spectrometry (TWIMS), Trapped Ion Mobility Spectrometry (TIMS), High Field Asymmetric waveform ion Mobility Spectrometry (FAIMS), Drift Tube Ion Mobility Spectrometry (DTIMS, Structures for Lossless Ion Manipulations (SLIM).'
+  constraints:
+    required: False
+    enum:
+      - TIMS
+      - TWIMS
+      - FAIMS
+      - DTIMS
+      - SLIM
+
+# include: ../includes/fields/data_collection_mode.yaml
+# include: ../includes/fields/ms_scan_mode.yaml
+# include: ../includes/fields/labeling.yaml
+# include: ../includes/v0/doi/section_prep.yaml
+
+- name: column_vendor
+  description: The manufacturer of the GC column
+- name: column_model
+  description: The model of the GC column used
+- name: column_length_value
+  description: GC column length
+- name: column_length_unit
+  description: Unit for GC column length (typically cm)
+  constraints:
+    enum:
+      - cm # TODO: Confirm?
```

Comment:
```suggestion
      - cm
```

Reply:
Column length for CE is cm and no one seems to respond to GC column length so cm is the safe option here.